docs/en/use_case/kafka_connector_demo.md

```diff
@@ -15,7 +15,7 @@ For OpenMLDB Kafka Connector implementation, please refer to [extensions/kafka-c
 
 ### Download and Preparation
 
-- Download Kafka: please click on [kafka_2.13-3.1.0](https://dlcdn.apache.org/kafka/3.1.0/kafka_2.13-3.1.0.tgz).
+- Download Kafka: please click [kafka downloads](https://kafka.apache.org/downloads) to download `kafka_2.13-3.1.0.tgz`.
```

Comment:
why not link the download url directly

Reply:
The download link may be invalid, so just use the download page, the address kafka.apache.org won't change